src/types/permissions.d.ts

```diff
@@ -30,3 +30,12 @@ export interface IdpGroup {
   name: string;
   groups: string[]; // these should be names of lxd groups
 }
+
+export interface LxdTrustToken {
```

Comment:
this type should be named `LxdFineGrainedTlsIdentity`, the token is what we will generate in the browser.